tests/integration/Integration.t.sol

```diff
@@ -24,6 +24,8 @@ abstract contract Integration_Test is Base_Test {
     // Common stream IDs to be used across the tests.
     // Default stream ID.
     uint256 internal defaultStreamId;
+    // A stream with a recipient contract that is not allowed to hook.
+    uint256 internal notAllowedtoHookStreamId;
```

Comment:
It's quite difficult to see that this is a stream ID (due to the length).

Should we create a struct or a contract so that we can do `streamIds.notAllowedToHook`, `streamIds.default`, etc.?

If yes, let's create an issue, please.

cc @sablier-labs/solidity

Reply:
Good idea.